components/GeoTriggering.js

```diff
@@ -85,6 +113,16 @@ export default function GeoTriggering() {
     <View style={styles.container}>
       <Text style={styles.eventTitle}>Geo Triggering</Text>
       {error ? <Text>Error: {error}</Text> : null}
+      <Text>Allow Background Location Updates</Text>
+      { Platform.OS == "ios" ? (
+        <Switch
+          onValueChange={toggleSwitch}
+          value={isBackgroundLocationUpdatesEnabled}
+        />
+        ): {}}
+      { Platform.OS == "ios" ? (
```

Comment:
Curious would this be any different from `if (Platform.OS === OS.IOS) {`

Reply:
I have just updated the Platform specific code for this. `if (Platform.OS === OS.IOS) {` seems to cause syntax error. So I'm just using the one from React native documentation